codalab/apps/teams/forms.py

```diff
@@ -1,14 +1,48 @@
 from django import forms
+from django.forms.widgets import ClearableFileInput, Input, CheckboxInput
+from django.utils.html import escape, conditional_escape
+from django.utils.encoding import force_unicode
 from .models import Team, TeamMembership
 from tinymce.widgets import TinyMCE
+from django.utils.html import format_html
+from django.utils.encoding import force_text
+from django.utils.safestring import mark_safe
 import os
 
+class CustomImageField(ClearableFileInput):
```

Comment:
Hmm, I mean within the file name.  Like can I name a file in such a way to inject a script tag? Probably not. 

Also just noticed `conditional_escape` being used... this code is awesome it looks like, thanks!